gems/faker/2.23/_test/test.rb

```diff
@@ -0,0 +1,18 @@
+# Write Ruby code to test the RBS.
+# It is type checked by `steep check` command.
+
+require "faker"
+
+Faker::Name.name + 'string'
```

Comment:
I wrote example codes in README.md https://github.com/faker-ruby/faker#usage
I'll create another PR when merged this pull request.